config/rbac/kustomization.yaml

```diff
@@ -11,3 +14,22 @@ resources:
 - auth_proxy_role_binding.yaml
 - auth_proxy_client_clusterrole.yaml
 
+# needed for nfd-worker
+# this patch is needed given that
+# +kubebuilder does not allow resourceNames
+patchesJSON6902:
+- target: 
+    kind: ClusterRole
+    name: manager-role
+  patch: |-
+    - op: add
+      path: /rules/1
+      value: 
+        apiGroups:
+        - policy
+        resources:
+        - podsecuritypolicies
+        verbs:
+        - use
+        resourceNames:
+        - nfd-worker
```

Comment:
Does this work as intended? Shouldn't you add new item to rules list (`path: /rules`) instead of modifying item 1 in the list?

Also, there's trailing whitespace on some lines.

Reply:
yup it works as expected
run `kustomize build config/default` and check the output 

trimmed the white spaces